lib/rules/no-invalid-position-at-import-rule/__tests__/index.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+const { messages, ruleName } = require('..');
+
+testRule({
+	ruleName,
+	config: [true],
+
+	accept: [
+		{
+			code: `
+			@import 'foo.css';
+			a {}
+			`,
+			description: '@import on first line',
+		},
+		{
+			code: `
+			/* some comment */
+			@import 'foo.css';
+			`,
+			description: '@import after comment',
+		},
+		{
+			code: `
+			@charset 'utf-8';
+			@import 'foo.css';
+			`,
+			description: '@import after @charset ',
+		},
+		{
+			code: `
+			@import 'foo.css';
+			@import 'bar.css';
+			`,
+			description: '@import after another @import',
+		},
+		{
+			code: `
+			@CHARSET 'utf-8';
+			@imPORT 'foo.css';
+			@import 'bar.css';
+			`,
+			description: 'case insensitive',
+		},
+	],
+
+	reject: [
```

Comment:
Let's add test for double message?

```
code: 'a {}; @import "foo.css"; @import "bar.css"',
warnings: [
	{
		message: messages.rejected(`foo.css`),
		description: '@import after selector',
	},
	{
		message: messages.rejected(`bar.css`),
		description: '@import after selector',
	}
]
```

Reply:
Let's also include `line` and `column` for each warning.